auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -27,19 +27,29 @@ class CheckPullRequest extends RequestHandler {
 
   final PubSub pubsub;
 
-  Future<Response> get() async {
-    final pub.PullResponse pullResponse = await pubsub.pull(1, 'auto-submit-queue-sub');
+  Future<List<Response>> get() async {
+    List<Response> responses = <Response>[];
+    final pub.PullResponse pullResponse = await pubsub.pull('auto-submit-queue-sub', 100);
```

Comment:
Avoid magic numbers. We should move 100 into something like `static const int kPullMesssageBatchSize = 100;`

Reply:
Thank you Casey for teaching me this, I will also take care to avoid the magi numbers in the future PRs.